examples/tenant-log-search.yaml

```diff
@@ -0,0 +1,103 @@
+
+## If a scheduler is specified here, Tenant pods will be dispatched by specified scheduler.
+## If not specified, the Tenant pods will be dispatched by default scheduler.
+# scheduler:
+#  name: my-custom-scheduler
+
+spec:
+  ## Registry location and Tag to download MinIO Server image
+  image: minio/minio:RELEASE.2020-10-18T21-54-12Z
+  imagePullPolicy: IfNotPresent
+
+  ## Secret with credentials to be used by MinIO Tenant.
+  ## Refers to the secret object created above.
+  credsSecret:
+    name: minio-creds-secret
+
+  ## Specification for MinIO Zone(s) in this Tenant.
+  zones:
+      ## Servers specifies the number of MinIO Tenant Pods / Servers in this zone.
+      ## For standalone mode, supply 1. For distributed mode, supply 4 or more.
+      ## Note that the operator does not support upgrading from standalone to distributed mode.
+    - servers: 4
+
+    ## volumesPerServer specifies the number of volumes attached per MinIO Tenant Pod / Server.
+      volumesPerServer: 4
+
+      ## This VolumeClaimTemplate is used across all the volumes provisioned for MinIO Tenant in this 
+      ## Zone.
+      volumeClaimTemplate:
+        metadata:
+          name: data
+        spec:
+          accessModes:
+            - ReadWriteOnce
+          resources:
+            requests:
+              storage: 1Ti
+
+
+  ## Mount path where PV will be mounted inside container(s).
+  mountPath: /export
+  ## Sub path inside Mount path where MinIO stores data.
+  # subPath: /data
+
+  ## Use this field to provide a list of Secrets with external certificates. This can be used to to configure
+  ## TLS for MinIO Tenant pods. Create secrets as explained here: 
+  ## https://github.com/minio/minio/tree/master/docs/tls/kubernetes#2-create-kubernetes-secret
+  # externalCertSecret:
+  #   - name: tls-ssl-minio
+  #     type: kubernetes.io/tls
+
+  ## Enable automatic Kubernetes based certificate generation and signing as explained in
+  ## https://kubernetes.io/docs/tasks/tls/managing-tls-in-a-cluster
+  requestAutoCert: true
+
+  ## Enable S3 specific features such as Bucket DNS which would allow `buckets` to be
+  ## accessible as DNS entries of form `<bucketname>.minio.default.svc.cluster.local`
+  s3:
+    ## This feature is turned off by default
+    bucketDNS: false
+
+  ## This field is used only when "requestAutoCert" is set to true. Use this field to set CommonName 
+  ## for the auto-generated certificate. Internal DNS name for the pod will be used if CommonName is 
+  ## not provided. DNS name format is *.minio.default.svc.cluster.local
+  certConfig:
+    commonName: ""
+    organizationName: []
+    dnsNames: []
+
+  ## PodManagement policy for MinIO Tenant Pods. Can be "OrderedReady" or "Parallel"
+  ## Refer https://kubernetes.io/docs/tutorials/stateful-application/basic-stateful-set/#pod-management-policy
+  ## for details.
+  podManagementPolicy: Parallel
+
+  log:
+    image: minio/logsearchapi:v3.0.28-14-gf0a7b7a
```

Comment:
The container image version will be determined based on the operator (released) version after https://github.com/minio/operator/pull/354 is merged. Until then, this is a placeholder.

Reply:
This is fixed now. `minio/logsearchapi` is now available in docker hub.